nucypher-core-wasm/src/lib.rs

```diff
@@ -601,6 +612,70 @@ impl ThresholdDecryptionRequest {
     }
 }
 
+//
+// E2EEThresholdDecryptionRequest
+//
+
+#[wasm_bindgen]
+#[derive(PartialEq, Debug, derive_more::From, derive_more::AsRef)]
+pub struct E2EEThresholdDecryptionRequest(nucypher_core::E2EEThresholdDecryptionRequest);
+
+#[wasm_bindgen]
+impl E2EEThresholdDecryptionRequest {
+    #[wasm_bindgen(js_name = fromBytes)]
+    pub fn from_bytes(data: &[u8]) -> Result<E2EEThresholdDecryptionRequest, Error> {
+        from_bytes::<_, nucypher_core::E2EEThresholdDecryptionRequest>(data)
+    }
+
+    #[wasm_bindgen(js_name = toBytes)]
+    pub fn to_bytes(&self) -> Box<[u8]> {
+        to_bytes(self)
+    }
+
+    #[wasm_bindgen(getter, js_name=decryption_request)]
+    pub fn decryption_request(&self) -> ThresholdDecryptionRequest {
+        ThresholdDecryptionRequest::from(self.0.decryption_request.clone())
+    }
+
+    #[wasm_bindgen(getter, js_name=responseEncryptingKey)]
+    pub fn response_encrypting_key(&self) -> PublicKey {
+        PublicKey::from(self.0.response_encrypting_key)
+    }
+}
+
+//
+// EncryptedThresholdDecryptionRequest
+//
+
+#[wasm_bindgen]
+#[derive(PartialEq, Debug, derive_more::From, derive_more::AsRef)]
+pub struct EncryptedThresholdDecryptionRequest(nucypher_core::EncryptedThresholdDecryptionRequest);
+
+#[wasm_bindgen]
+impl EncryptedThresholdDecryptionRequest {
+    #[wasm_bindgen(getter)]
+    pub fn id(&self) -> u16 {
```

Comment:
Better `ritural_id` (`ritualId` in JS? or is it `ritualID`?) to match the Rust API

Reply:
✔️